src/ripple/net/impl/HTTPStream.cpp

```diff
@@ -49,8 +49,8 @@ SSLStream::connect(
 
     boost::system::error_code ec;
 
-    auto fail = [&errorOut](std::string const& errorIn) {
-        errorOut = errorIn;
+    auto fail = [&errorOut, &ec](std::string const& errorIn) {
+        errorOut = errorIn + ": " + ec.message();
```

Comment:
If ec is not set, we append ": ". It may be a good ide to check ec before appending it.
eg.
```
        errorOut = errorIn;
        if (ec)
            errorOut += ". Error code: " + ec.message();
```

Reply:
Got this done with an additional defaulted parameter to the lambda